src/item_contents.cpp

```diff
@@ -307,10 +307,12 @@ ret_val<item_pocket *> item_contents::find_pocket_for( const item &it,
         item_pocket::pocket_type pk_type )
 {
     static item_pocket *null_pocket = nullptr;
-    ret_val<item_pocket *> ret = ret_val<item_pocket *>::make_failure( null_pocket,
-                                 _( "is not a container" ) );
+    std::string msg = "is not a container";
```

Comment:
This needs to 
```suggestion
    std::string msg = _( "is not a container" );
```
to be translated.
You can figure something else out for the comparison you do below.